OpenRA.Mods.Common/AI/LimitUnitByActors.cs

```diff
@@ -1,4 +1,15 @@
-﻿using System;
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
```

Comment:
2018

Reply:
OK I'll make sure these are and stay 2018 over the main files. :)

Reply:
please do not remove this ```using System``` it is important for travis